documentation/topics/security/policies.md

```diff
@@ -54,6 +54,37 @@ There are four check types, all of which do what they sound like they do:
 
 If a single check does not explicitly authorize or forbid the whole policy, then the flow moves to the next check. For example, if an `authorize_if` check does NOT return true, this _does not mean the whole policy is forbidden_ - it means that further checking is required.
 
+### Alternative Ways To Write a Policy
+
+#### Policy with keyword list
+
+```elixir
+policies do
+  policy always(), authorize_if: always()
+end
+```
+
+#### Policy with `condition` inside `do` block
+
+```elixir
+policies do
+  policy do
+    condition always()
```

Comment:
I was not aware of `condition`. I have discovered it in one of the [tests](https://github.com/ash-project/ash/blob/cef5900f40bed52b26a261ed30908faf3d016286/test/actions/aggregate_test.exs#L49).